### DISCUSSION TITLE
githup Airdrop Kazandım Puanlarım Verilmedi .

### DISCUSSION THREAD

**Discussion by Despair016** [IBAX-io/go-ibax#1216] [Soru Cevap - Türkçe] [answered]
@BadAssKyle @akibax @ibaxio 
Tweeter Gıthup Airdrop Kazananlarındanım Lakın puanlarım Verilmedi.
@Inter_student kullanıcı nıckım  Gerekli Talimatları  @IbaxNetwork'a  Tweeterden Gönderdım Hala Okunmadı.

**Comment by Yusufknk** [comment, marked as answer]
Ödüller ve hesaplar inceleme yapıldıktan sonra dağıtılacaktır. Lütfen sabırlı olun.

**Reply by Despair016** [reply]
üzerinden 4 güm geçti kazanan kişi sayısıda cok degil cok gecikmedımı

**Reply by Yusufknk** [reply]
Haklısınız son çekilişte gecikme yaşandı. Ama lütfen sabırla bekleyin.

**Reply by Despair016** [reply]
> Haklısınız son çekilişte gecikme yaşandı. Ama lütfen sabırla bekleyin.

bütün yöneticilere Yazdım hala olumlu dönüş yapan olmadı .
tweter Daki Dm Yi okumaları bu kdar zor olmamalı :(

**Reply by Despair016** [reply]
sesimi duyan yokmu yöneticilerden ????

**Reply by Yusufknk** [reply]
Ödüller dağıtılacaktır arkadaşlar lütfen bekleyin.

**Reply by Despair016** [reply]
İnş Bu Geciken Hatayı Güzel Bir Telafi İle Yaparlar Toplam puanum 32 k yı gececekti ne güze :(
